### DISCUSSION TITLE
Redbean 2.0 ❤️ Docker

### DISCUSSION THREAD

**Discussion by kissgyorgy** [jart/cosmopolitan#458] [Show and tell]
Still awesome project! 😊
I updated my Dockerfiles to work with Redbean 2.0: https://github.com/kissgyorgy/redbean-docker

**Comment by jart** [comment]
Awesome to hear!

**Comment by jart** [comment]
Consider joining our Discord. https://discord.gg/thZ9mVu9 Everyone is going nuts over your latest update.
